src/pf-modal/index.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-modal example</title>
+    <link rel="stylesheet" href="../../dist/css/patternfly.css">
+    <link rel="stylesheet" href="../../dist/css/patternfly-additions.css">
+    <link rel="stylesheet" href="../../dist/css/patternfly-webcomponents.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="../../dist/js/patternfly.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+      <h1>Modals</h1>
+    </div>
+    <hr>
```

Comment:
@dabeng  Can we remove this `<hr>` ? It's superfluous after the `page-header` div (and shows borders instead of one). Otherwise...i think this looks great. Nice work!